<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>331</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>331</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>331</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="memberList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editChatNameEditLine">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>更改聊天室名称</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editChatNameBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>更改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addEditLine">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>添加成员 请输入ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="delEditLine">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>删除成员 请输入ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leaveRoomBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>296</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>解散聊天室</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
